packages/wrangler/src/config/environment.ts

```diff
@@ -877,4 +877,11 @@ export type ExperimentalAssets = {
 	/** Absolute path to assets directory */
 	directory: string;
 	binding?: string;
+	serve_exact_matches_only?: boolean;
+	trailing_slashes?: "auto" | "add" | "remove";
+	not_found_behavior?:
+		| "default"
+		| "single-page-application"
+		| "404-page"
```

Comment:
Do we need to distinguish between this as nearest?
If the use wants to always use the top level 404 then they should just avoid adding 404.html files elsewhere.